.github/workflows/ci.yml

```diff
@@ -118,6 +118,9 @@ jobs:
     - name: Test custom prng
       run: |
         JAX_ENABLE_CUSTOM_PRNG=1 pytest -vs test/infer/test_mcmc.py
+    - name: Test nested sampling
+      run: |
+        JAX_ENABLE_X64=1 pytest -vs test/contrib/test_nested_sampling.py
```

Comment:
you can ignore nested sampling at line 1078

Reply:
I am not sure which line you are talking about here. I made sure that the `jaxns` related tests are skipped if double precision is not enabled.